global.json

```diff
@@ -1,7 +1,7 @@
 {
   "sdk": {
     "version": "8.0.100",
-    "rollForward": "latestMajor",
+    "rollForward": "latestMinor",
```

Comment:
Not directly, but the CI tests crashed out, and I had assumed it might be due to a new SDK being picked up since .NET 9 was released. I'm no longer sure that's the issue. Investigating now.

Reply:
The latest commit, starting with a new SDK version, seems to be working, so this is fine to include for now, and we'll readdress with a larger piece of work to support .NET 9 SDK.